src/List.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import { addDoc } from 'firebase/firestore';
+import db from './firebase';
+import { collection, onSnapshot } from 'firebase/firestore';
+
+export default function Db() {
+  const [docs, setDocs] = useState([]);
+  useEffect(() => {
```

Comment:
I have some thoughts around how we're interacting with firebase here. I see this is approach that's been recommended via the issue itself.. imo this is the ["JavaScript" way](https://firebase.google.com/docs/firestore/query-data/listen) of doing it where [there is a more "React" way](https://github.com/CSFrequency/react-firebase-hooks/tree/v4.0.x/firestore#usecollection) of communicating with Firebase that I think will be better suited for our app. With this approach, we're using the standard firebase JS library and effectively writing our own hook every time we wish to interact with firebase in this way. We can skip this step by taking advantage of a[ common Firebase library:](https://github.com/CSFrequency/react-firebase-hooks/tree/v4.0.x/firestore) that is written with React developers in mind. It exposes a function that does what we're doing here with a bit less hassle. You can read up on it [here](https://github.com/CSFrequency/react-firebase-hooks/tree/v4.0.x/firestore#usecollection)

Reply:
Thank you for this feedback! Checking it out.

Reply:
@tannaurus Thank you for sharing this, that makes a lot of sense!

@danainjax Since issue 4 deals with submitting to the database, I can make this change with Dev if you don't mind. Let me know your thoughts!

Reply:
Sounds great, Viviana. I tried to wrap my brain around it this morning and got stuck. Hoping I can make office hours this week and talk through it. Feel free to proceed. 👍🏻